src/SFA.DAS.ProviderApprenticeshipsService.Infrastructure/Services/HashingService.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace SFA.DAS.ProviderApprenticeshipsService.Infrastructure.Services
+{
+    using HashidsNet;
+    using SFA.DAS.ProviderApprenticeshipsService.Domain.Interfaces;
+    using SFA.DAS.ProviderApprenticeshipsService.Infrastructure.Configuration;
+
+    public class HashingService : IHashingService
+    {
+        private readonly Hashids _hashIds;
+        private const string Hashstring = "SFA: digital apprenticeship service";
+        private const string AllowedCharacters = "46789BCDFGHJKLMNPRSTVWXY";
+
+        public HashingService(ProviderApprenticeshipsServiceConfiguration configuration)
+        {
+            var hashstring = string.IsNullOrEmpty(configuration.Hashstring)
```

Comment:
would be simpler if the ctor just accepted a string rather than the config type. this way it will be easier to refactor to a separate class/package